MAINTAINERS

```diff
@@ -2,6 +2,7 @@
   "version": 1,
   "file_format": "This MAINTAINERS file format is described at https://github.com/puppetlabs/maintainers",
   "issues": "https://github.com/puppetlabs/clj-puppetdb/issues",
+  "internal_list": "https://groups.google.com/a/puppet.com/forum/?hl=en#!forum/discuss-console-ui-maintainers",
```

Comment:
@scottyw care to weigh in? I'm agnostic about what the list actually is, as long as it's in this file (let me know if you want a new one created so we can update IT)